client/my-sites/plugins/plugin.jsx

```diff
@@ -351,19 +356,24 @@ const SinglePlugin = createReactClass( {
 export default connect(
 	( state, props ) => {
 		const selectedSiteId = getSelectedSiteId( state );
+		const sites = getSelectedOrAllSitesWithPlugins( state );
+
+		// eslint-disable-next-line wpcalypso/redux-no-bound-selectors
+		const siteIds = uniq( sites ).map( ( site ) => site.ID );
```

Comment:
nit-question: Should this live in selector space? Is `uniq( sites.map( ( site ) => site.ID ) )` maybe even cheaper?

Reply:
> Is uniq( sites.map( ( site ) => site.ID ) ) maybe even cheaper?

Yes 👍  Improved in ace7173ea565e3b91522d72cdda2cada22df12e6

> Should this live in selector space?

Yes. I'll make a note to follow-up in another PR 👍